src/arch/armv8/asm_defs.c

```diff
@@ -16,13 +17,23 @@ __attribute__((used)) static void cpu_defines(void)
     DEFINE_SIZE(CPU_STACK_SIZE, ((struct cpu*)NULL)->stack);
 
     DEFINE_OFFSET(CPU_VCPU_OFF, struct cpu, vcpu);
+
+#ifndef MEM_PROT_MMU
+    DEFINE_OFFSET(CPU_AS_ARCH_MASK_OFF, struct cpu, as.arch.entry_mask);
+    DEFINE_OFFSET(CPU_ARCH_PROFILE_MPU_LOCKED_OFF, struct cpu, arch.profile.mpu.locked);
+#endif
```

Comment:
```suggestion
#endif /* MEM_PROT_MPU */
```

Reply:
Fixed in c26680d8c6a652fe312b41c9c5799c45aab308ce